src/test/java/gov/cancer/tests/crosscutting/RelatedResources_Test.java

```diff
@@ -0,0 +1,105 @@
+package gov.cancer.tests.crosscutting;
+
+import java.util.Iterator;
+import java.util.List;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import gov.cancer.framework.ExcelDataReader;
+import gov.cancer.pageobject.crosscutting.RelatedResourcesPage;
+import gov.cancer.pageobject.helper.RelatedResource;
+import gov.cancer.tests.TestObjectBase;
+import gov.cancer.tests.TestRunner;
+
+public class RelatedResources_Test extends TestObjectBase {
+
+  @Test(dataProvider = "getPagesWithRelatedResources")
+  public void VerifyRelatedResourcesSectionAppears(String path) {
+
+    // Get the page.
+    TestRunner.run(RelatedResourcesPage.class, path, (RelatedResourcesPage page) -> {
+
+      // Assert the related resources section is visible.
+      Assert.assertTrue(page.hasRelatedResources(), "Related Resources section is visible.");
+
+    });
+  }
+
+ 
+
```

Comment:
Please add a comment block. What is this testing? What are the inputs?  Make things easier for the person reading this six months from now.

Reply:
I will add comment block to methods.